src/Stryker.Core/Stryker.Core.UnitTest/Reporters/ConsoleReportReporterTests.cs

```diff
@@ -230,8 +229,7 @@ public void ConsoleReportReporter_ShouldGreenOn80PercentAndHigher()
                 Type = MutatorType.Arithmetic
             };
 
-            var target = new ConsoleReportReporter(new StrykerOptions("", "ReportOnly", "", 1000, null, "debug", false, false, 1, 80, 60, 0), chalkMock.Object);
-
+            var target = new ConsoleReportReporter(new StrykerOptions(), chalkMock.Object);
```

Comment:
Since no values are passed to StrykerOptions for this test, I suggest changing the test name to `ConsoleReportReporter_ShouldPrintGreenAboveThresholdHigh`